### ISSUE TITLE
Is it possible to replicate pose estimation model? (train details inside)

### ISSUE THREAD

**Issue by chenlize96** [google/mediapipe#3310]
Since we don't have the interior code for training part, I created a model which refers to pose_landmark_full.tflite. My_model.tflite is the same as pose_landmark_full.tflite when I use netron.app to view.
optimizer = tf.keras.optimizers.Adam(learning_rate=0.000001)
loss = tf.keras.losses.MeanSquaredError()
metrics=['accuracy']
These are my train variables.
My dataset only contains 1300 images.
Epoch 1/50
1309/1309 [==============================] - 47s 24ms/step - loss: 17136.2461 - accuracy: 0.5134 - val_loss: 15538.5098 - val_accuracy: 0.5384
Epoch 2/50
1309/1309 [==============================] - 28s 21ms/step - loss: 17098.1934 - accuracy: 0.5136 - val_loss: 15476.3525 - val_accuracy: 0.5384
Epoch 3/50
1309/1309 [==============================] - 28s 22ms/step - loss: 16984.5742 - accuracy: 0.5141 - val_loss: 15301.8701 - val_accuracy: 0.5490
Epoch 4/50
1309/1309 [==============================] - 28s 22ms/step - loss: 16514.1641 - accuracy: 0.5207 - val_loss: 14418.5303 - val_accuracy: 0.5490
Epoch 5/50
1309/1309 [==============================] - 28s 22ms/step - loss: 15017.6338 - accuracy: 0.5249 - val_loss: 12381.3535 - val_accuracy: 0.5762
Epoch 6/50
1309/1309 [==============================] - 30s 23ms/step - loss: 11778.9180 - accuracy: 0.5266 - val_loss: 8166.1133 - val_accuracy: 0.5495
Epoch 7/50
1309/1309 [==============================] - 28s 21ms/step - loss: 6379.1235 - accuracy: 0.5415 - val_loss: 3126.3193 - val_accuracy: 0.5762
Epoch 8/50
1309/1309 [==============================] - 28s 22ms/step - loss: 2603.0703 - accuracy: 0.5711 - val_loss: 1844.6799 - val_accuracy: 0.6796
Epoch 9/50
1309/1309 [==============================] - 28s 22ms/step - loss: 1961.0652 - accuracy: 0.6449 - val_loss: 1668.4869 - val_accuracy: 0.7337
Epoch 10/50
1309/1309 [==============================] - 28s 22ms/step - loss: 1800.4517 - accuracy: 0.6789 - val_loss: 1540.9895 - val_accuracy: 0.7920
Epoch 11/50
1309/1309 [==============================] - 28s 22ms/step - loss: 1692.8059 - accuracy: 0.6909 - val_loss: 1449.9126 - val_accuracy: 0.7920
Epoch 12/50
1309/1309 [==============================] - 28s 21ms/step - loss: 1620.9713 - accuracy: 0.6966 - val_loss: 1386.2838 - val_accuracy: 0.7992
Epoch 13/50
1309/1309 [==============================] - 28s 22ms/step - loss: 1575.0228 - accuracy: 0.6997 - val_loss: 1336.3586 - val_accuracy: 0.7992
Epoch 14/50
1309/1309 [==============================] - 28s 22ms/step - loss: 1546.4478 - accuracy: 0.6997 - val_loss: 1309.5891 - val_accuracy: 0.7992
Epoch 15/50
1309/1309 [==============================] - 28s 22ms/step - loss: 1529.7327 - accuracy: 0.6997 - val_loss: 1287.0593 - val_accuracy: 0.7992
Epoch 16/50
1309/1309 [==============================] - 28s 21ms/step - loss: 1520.2056 - accuracy: 0.6997 - val_loss: 1276.5696 - val_accuracy: 0.7992
Epoch 17/50
1309/1309 [==============================] - 28s 22ms/step - loss: 1515.2650 - accuracy: 0.6997 - val_loss: 1272.1436 - val_accuracy: 0.7992
Epoch 18/50
1309/1309 [==============================] - 28s 22ms/step - loss: 1512.7478 - accuracy: 0.6997 - val_loss: 1261.3700 - val_accuracy: 0.7992
Epoch 19/50
1309/1309 [==============================] - 30s 23ms/step - loss: 1511.3035 - accuracy: 0.6997 - val_loss: 1261.7997 - val_accuracy: 0.7992
Epoch 20/50
1309/1309 [==============================] - 29s 22ms/step - loss: 1509.9744 - accuracy: 0.6997 - val_loss: 1250.6017 - val_accuracy: 0.7992
Epoch 21/50
1309/1309 [==============================] - 30s 23ms/step - loss: 1510.2988 - accuracy: 0.6997 - val_loss: 1256.1412 - val_accuracy: 0.7992
Epoch 22/50
1309/1309 [==============================] - 28s 21ms/step - loss: 1510.0756 - accuracy: 0.6997 - val_loss: 1255.4430 - val_accuracy: 0.7992
Epoch 23/50
1309/1309 [==============================] - 30s 23ms/step - loss: 1509.9825 - accuracy: 0.6997 - val_loss: 1258.1820 - val_accuracy: 0.7992
Epoch 24/50
1309/1309 [==============================] - 28s 21ms/step - loss: 1509.4059 - accuracy: 0.6997 - val_loss: 1249.2114 - val_accuracy: 0.7992
..................................................................................
Epoch 50/50
1309/1309 [==============================] - 28s 22ms/step - loss: 1509.5070 - accuracy: 0.6997 - val_loss: 1253.9976 - val_accuracy: 0.7992

Here is my result. I tried to test my model - it doesn't work since different inputs gave me a same output.
I am confused why this happens and how to solve it? Can anyone tell me if it is possible to replicate pose estimation model?
Thanks.

**Comment by sureshdagooglecom** [comment]
Hi @chenlize96 ,
I can clearly see that you are not using MediaPipe solutions here , you are using tensorflow to train your model ,which we don't support for that in this platform , kindly revert back if any issues while using core MediaPipe solutions.

**Comment by sainisanjay** [comment]
> Since we don't have the interior code for training part, I created a model which refers to pose_landmark_full.tflite. My_model.tflite is the same as pose_landmark_full.tflite when I use netron.app to view. optimizer = tf.keras.optimizers.Adam(learning_rate=0.000001) loss = tf.keras.losses.MeanSquaredError() metrics=['accuracy'] These are my train variables. My dataset only contains 1300 images. Epoch 1/50 1309/1309 [==============================] - 47s 24ms/step - loss: 17136.2461 - accuracy: 0.5134 - val_loss: 15538.5098 - val_accuracy: 0.5384 Epoch 2/50 1309/1309 [==============================] - 28s 21ms/step - loss: 17098.1934 - accuracy: 0.5136 - val_loss: 15476.3525 - val_accuracy: 0.5384 Epoch 3/50 1309/1309 [==============================] - 28s 22ms/step - loss: 16984.5742 - accuracy: 0.5141 - val_loss: 15301.8701 - val_accuracy: 0.5490 Epoch 4/50 1309/1309 [==============================] - 28s 22ms/step - loss: 16514.1641 - accuracy: 0.5207 - val_loss: 14418.5303 - val_accuracy: 0.5490 Epoch 5/50 1309/1309 [==============================] - 28s 22ms/step - loss: 15017.6338 - accuracy: 0.5249 - val_loss: 12381.3535 - val_accuracy: 0.5762 Epoch 6/50 1309/1309 [==============================] - 30s 23ms/step - loss: 11778.9180 - accuracy: 0.5266 - val_loss: 8166.1133 - val_accuracy: 0.5495 Epoch 7/50 1309/1309 [==============================] - 28s 21ms/step - loss: 6379.1235 - accuracy: 0.5415 - val_loss: 3126.3193 - val_accuracy: 0.5762 Epoch 8/50 1309/1309 [==============================] - 28s 22ms/step - loss: 2603.0703 - accuracy: 0.5711 - val_loss: 1844.6799 - val_accuracy: 0.6796 Epoch 9/50 1309/1309 [==============================] - 28s 22ms/step - loss: 1961.0652 - accuracy: 0.6449 - val_loss: 1668.4869 - val_accuracy: 0.7337 Epoch 10/50 1309/1309 [==============================] - 28s 22ms/step - loss: 1800.4517 - accuracy: 0.6789 - val_loss: 1540.9895 - val_accuracy: 0.7920 Epoch 11/50 1309/1309 [==============================] - 28s 22ms/step - loss: 1692.8059 - accuracy: 0.6909 - val_loss: 1449.9126 - val_accuracy: 0.7920 Epoch 12/50 1309/1309 [==============================] - 28s 21ms/step - loss: 1620.9713 - accuracy: 0.6966 - val_loss: 1386.2838 - val_accuracy: 0.7992 Epoch 13/50 1309/1309 [==============================] - 28s 22ms/step - loss: 1575.0228 - accuracy: 0.6997 - val_loss: 1336.3586 - val_accuracy: 0.7992 Epoch 14/50 1309/1309 [==============================] - 28s 22ms/step - loss: 1546.4478 - accuracy: 0.6997 - val_loss: 1309.5891 - val_accuracy: 0.7992 Epoch 15/50 1309/1309 [==============================] - 28s 22ms/step - loss: 1529.7327 - accuracy: 0.6997 - val_loss: 1287.0593 - val_accuracy: 0.7992 Epoch 16/50 1309/1309 [==============================] - 28s 21ms/step - loss: 1520.2056 - accuracy: 0.6997 - val_loss: 1276.5696 - val_accuracy: 0.7992 Epoch 17/50 1309/1309 [==============================] - 28s 22ms/step - loss: 1515.2650 - accuracy: 0.6997 - val_loss: 1272.1436 - val_accuracy: 0.7992 Epoch 18/50 1309/1309 [==============================] - 28s 22ms/step - loss: 1512.7478 - accuracy: 0.6997 - val_loss: 1261.3700 - val_accuracy: 0.7992 Epoch 19/50 1309/1309 [==============================] - 30s 23ms/step - loss: 1511.3035 - accuracy: 0.6997 - val_loss: 1261.7997 - val_accuracy: 0.7992 Epoch 20/50 1309/1309 [==============================] - 29s 22ms/step - loss: 1509.9744 - accuracy: 0.6997 - val_loss: 1250.6017 - val_accuracy: 0.7992 Epoch 21/50 1309/1309 [==============================] - 30s 23ms/step - loss: 1510.2988 - accuracy: 0.6997 - val_loss: 1256.1412 - val_accuracy: 0.7992 Epoch 22/50 1309/1309 [==============================] - 28s 21ms/step - loss: 1510.0756 - accuracy: 0.6997 - val_loss: 1255.4430 - val_accuracy: 0.7992 Epoch 23/50 1309/1309 [==============================] - 30s 23ms/step - loss: 1509.9825 - accuracy: 0.6997 - val_loss: 1258.1820 - val_accuracy: 0.7992 Epoch 24/50 1309/1309 [==============================] - 28s 21ms/step - loss: 1509.4059 - accuracy: 0.6997 - val_loss: 1249.2114 - val_accuracy: 0.7992 .................................................................................. Epoch 50/50 1309/1309 [==============================] - 28s 22ms/step - loss: 1509.5070 - accuracy: 0.6997 - val_loss: 1253.9976 - val_accuracy: 0.7992
> 
> Here is my result. I tried to test my model - it doesn't work since different inputs gave me a same output. I am confused why this happens and how to solve it? Can anyone tell me if it is possible to replicate pose estimation model? Thanks.

@chenlize96  Is it possible for you to share the "pose_landmark_full.tflite" model architecture? I am just wonder how you able to  create the model.
